nss-day-cohort-60/national-parks-client
# Client can now display a list of national parks

# Description
Created a ParksList and Park module to enable the client to be able to display all national parks in a list 

# Steps to test

1. Fetch the **parks-list** branch and switch to it
2. Start debugger in API
3. Start national-parks-client React app 
4. Verify list of national parks displays on landing page
